app/src/main/java/org/mozilla/fenix/settings/address/view/AddressEditorView.kt

```diff
@@ -25,6 +28,11 @@ internal const val DEFAULT_COUNTRY = "US"
 
 /**
  * Shows an address editor for adding or updating an address.
+ *
+ * @param binding The binding used to display the view.
+ * @param interactor Handles view interactions.
+ * @param region If available, will automatically set the country dropdown if adding a new address.
```

Comment:
Are we posting the documentation anywhere using a tool like that wouldn't link the type next to the parameter name when it lists the function signature? Just curious why there's a precedent of including the type in the kDoc. The tools I've used to generate HTML pages for them before usually include the type right next to the parameter, and it's not a far stretch to access the type within the code itself. Mostly seems like an additional opportunity for the docs to get out of sync.

Reply:
This question is above my pay grade lol. @gabrielluong do you have any insight to the history of our kdocs?

Reply:
I don't think we are generating HTML pages for our docs. I didn't really think much about posting the function signature at the front here because it seemed to me it was being used as part of the sentence so I didn't really see anything wrong. If the intent was to post the literal function signature at the very beginning of everything, then I would probably say avoid that. When it comes to kDoc, I will typically defer to what I see is common in AC since that project has been richly documented due to the intent of project to be a public platform that would serve other projects to build browsers based on the libraries that are provided. Otherwise, I would prefer if we could adopt a style guide that is publicly available with appropriate linting to enforce. I don't know if there is such a thing for kDocs and Kotlin that is widely recognize across multiple organizations. An example would be something like https://google.github.io/styleguide/pyguide.html, https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html, https://prettier.io/

Reply:
Okay, I'll take the suggestions here. It sounds like, in general, there's no need to specify the `@param` type in its description in every case though?